task14/index.js

```diff
@@ -0,0 +1,109 @@
+const parent = document.getElementById("js-parent"); 
+const modalPlace = document.getElementById('modalPlace');
+
+const appendModalForParent = () => {
+    const modalText = document.createElement('p');
+    const getNumber = document.createElement('input')
+    const closeBtn = document.createElement('input');
+    modalPlace.className = "modal";
+    modalPlace.style.display = "block";
+    modalText.textContent = "数字を入力して値を取得します。";
+    getNumber.id = "js-number";
+    getNumber.className = "number";
+    getNumber.type = "number";
+    closeBtn.id = "js-closeBtn";
+    closeBtn.className = "close-button";
+    closeBtn.type = "button";
+    closeBtn.value = "click";
+
+    modalPlace.appendChild(modalText);
+    modalPlace.appendChild(getNumber);
+    modalPlace.appendChild(closeBtn);
+};
+
+const appendButtonForParent = () => {
+    const button = document.createElement('input');
+    button.id = "js-openBtn";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+appendButtonForParent();
+
+const requestNumber = () => {
+    const getValue = document.getElementById("js-number");
+    if(getValue.value === ""){
+        alert("数字を入力してください");
+        return false;
+    } else {
+        console.log(getValue.value);
+        modalPlace.style.display = "none";
+        init();
+        return true;
+    }
+};
+
+const openBtn = document.getElementById("js-openBtn");
+openBtn.addEventListener("click", (e) => {
+    appendModalForParent();
+    e.target.remove();
+    modalClose();
+});
+
+const modalClose = () => {
+    const closeBtn = document.getElementById("js-closeBtn");
+    closeBtn.addEventListener("click", () => {
+        requestNumber();
+    });
+};
+
+const getFetchData = async () => {
+    const response = await fetch("https://jsondata.okiba.me/v1/json/3jhPW210812052923");
+    const json = await response.json();
+    return json.data;
+};
+
+const loading = () => {
```

Comment:
IMO
How about adding "add" as in removeLoading?
`addLoading`

Reply:
It certainly looks more beautiful that way.
I'm a naming lost cause, so I'm learning!
a8467f1